### PR TITLE
Update policy-csp-admx-sam.md

### PR DESCRIPTION
**Pull request by RAJU2529**
as per user report issue #11671  so i added a new link

**https://support.microsoft.com/windows/update-your-security-processor-tpm-firmware-94205cbc-a492-8d79-cc55-1ecd6b0a8022**

### CODE REVIEW

File: windows/client-management/mdm/policy-csp-admx-sam.md [hunk 1]
```diff
@@ -49,21 +49,21 @@ For more information on the ROCA vulnerability, please see:
...
 
 If not configured, domain controllers will default to using their local configuration. The default local configuration is Audit.
 
 A reboot isn't required for changes to this setting to take effect.
 
-Note to avoid unexpected disruptions this setting shouldn't be set to Block until appropriate mitigations have been performed, for example patching of vulnerable TPMs.
+Note to avoid unexpected disruptions this setting shouldn't be set to Block until appropriate mitigations have been performed, for example, patching of vulnerable TPMs.
 
-More information is available at< https://go.microsoft.com/fwlink/?linkid=2116430>.
+More information is available at [Update your (TPM) firmware](https://support.microsoft.com/windows/update-your-security-processor-tpm-firmware-94205cbc-a492-8d79-cc55-1ecd6b0a8022)
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/policy-csp-admx-sam.md:66]:
```suggestion
For more information, see [Update your (TPM) firmware](https://support.microsoft.com/windows/update-your-security-processor-tpm-firmware-94205cbc-a492-8d79-cc55-1ecd6b0a8022).
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by aczechowski** [comment]
Hi @RAJU2529 thanks for the edit, but it needs to be in one of the editable regions of the CSP article (which is now mostly automated). For more info, see [Contributing to the CSP reference articles](https://learn.microsoft.com/windows/client-management/mdm/contribute-csp-reference). Let me know if you have any questions.

Realistically that's just a matter of moving it down a couple of lines 😉

**Comment by RAJU2529** [comment]
@aczechowski , I added only link, I think I not moved any lines

**Comment by aczechowski** [comment]
@RAJU2529 

1. Please read the instructional article I linked previously. 😄 [Contributing to the CSP reference articles](https://learn.microsoft.com/en-us/windows/client-management/mdm/contribute-csp-reference) - it explains all of this.

2. I took a closer look - the link in question already exists in the non-editable description. So if I accept your change, it will revert back in a week or so when the automation reruns. There are two options:

  1. Close this PR and the linked issue, the change won't persist.
  2. Revise this PR to not edit line 66 (which comes from the ADMX), but add a new line in the editable section after line 69. It will effectively be redundant to what's on line 66, but a functional link.

Regardless, the original poster should [Send feedback to Microsoft with the Feedback Hub app](https://support.microsoft.com/windows/send-feedback-to-microsoft-with-the-feedback-hub-app-f59187f8-8739-22d6-ba93-f66612949332). Making sure to note that the string needs to change in the source ADMX, which is where that policy description comes from. That feedback should go to the engineering team who would need to file a bug to fix the string in the ADMX code.

I can also fix fwlink 2116430 so that it points to the article you found, so that will help as well. (I'll comment some of this back on the original issue as well. Vinay is out right now, so I don't know if he had something else in mind...)

**Comment by RAJU2529** [comment]
@aczechowski , Thank you sir for your points. I  understood, that editing of CSP article little bit difficult without knowing the background.
Closing from my end here, just now I read **https://learn.microsoft.com/en-us/windows/client-management/mdm/contribute-csp-reference**

**Comment by RAJU2529** [comment]
@aczechowski    Closing

**Pull request closed without merging**